homeassistant/components/music_assistant/__init__.py

```diff
@@ -0,0 +1,162 @@
+"""Music Assistant (music-assistant.io) integration."""
+
+from __future__ import annotations
+
+import asyncio
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
+
+from music_assistant.client import MusicAssistantClient
+from music_assistant.client.exceptions import CannotConnect, InvalidServerVersion
+from music_assistant.common.models.enums import EventType
+from music_assistant.common.models.errors import MusicAssistantError
+
+from homeassistant.config_entries import ConfigEntry, ConfigEntryState
+from homeassistant.const import CONF_URL, EVENT_HOMEASSISTANT_STOP, Platform
+from homeassistant.core import Event, HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.issue_registry import (
+    IssueSeverity,
+    async_create_issue,
+    async_delete_issue,
+)
+
+from .const import DOMAIN, LOGGER
+
+if TYPE_CHECKING:
+    from music_assistant.common.models.event import MassEvent
+
+type MusicAssistantConfigEntry = ConfigEntry[MusicAssistantEntryData]
+
+PLATFORMS = [Platform.MEDIA_PLAYER]
```

Comment:
```suggestion
PLATFORMS = (Platform.MEDIA_PLAYER,)
```

Reply:
what extra checks ?
There is no need for the added overhead of a list here imo, a tuple or set is just fine.
So enlighten me, what am I missing ?

Reply:
There are pylint checks that check if they are correct and alphabetic. Afaik that only works with lists and that's something I rather keep consistent for the sake of consistency. If we decide that this indeed causes more overhead for the project, I think we should change this everywhere, but that's a discussion apart from this PR

Reply:
Discussed this in discord: while it is true that this should probably be a tuple everywhere, to keep consistency, use a list here now. Imo the pylint check is just being dumb if it can not check the order of a tuple ;-)